lib/Sema/CSSolver.cpp

```diff
@@ -1470,7 +1475,25 @@ ConstraintSystem::solve(SyntacticElementTarget &target,
         return std::move(result);
       }
 
+      auto solutionsRef = std::move(solution).takeAmbiguousSolutions();
+      SmallVector<Solution, 4> ambiguity(
+          std::make_move_iterator(solutionsRef.begin()),
+          std::make_move_iterator(solutionsRef.end()));
+
+      {
+        SolverState state(*this, FreeTypeVariableBinding::Disallow);
+
+        // Constraint generator is allowed to introduce fixed to the
```

Comment:
Typo "introduce fixed to the" -> "introduce fixed solutions to the"? =]

Reply:
Oops, thank you!

Reply:
I think I misunderstood initially, but now it makes more sense :)

Reply:
How much difference does one character make :)